datasets/flwr_datasets/common/telemetry.py

```diff
@@ -0,0 +1,222 @@
+# Copyright 2023 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Flower telemetry."""
+
+
+import datetime
+import json
+import logging
+import os
+import platform
+import urllib.request
+import uuid
+from concurrent.futures import Future, ThreadPoolExecutor
+from enum import Enum, auto
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Union, cast
+
+from flwr_datasets.common.version import package_name, package_version
+
+FLWR_TELEMETRY_ENABLED = os.getenv("FLWR_TELEMETRY_ENABLED", "1")
+FLWR_TELEMETRY_LOGGING = os.getenv("FLWR_TELEMETRY_LOGGING", "0")
```

Comment:
Note that the names of environment variables are exactly the same as in Flower. Please confirm it's desired.

Reply:
Yes, this is correct. It allows someone who wants to disable telemetry to disable it for both the Flower Framework and for Flower Datasets.